dataset_generation/hi_wikisplit/Dataset.py

```diff
@@ -0,0 +1,218 @@
+import os
+import logging
+import sys
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+class RawData:
+    def __init__(self, input_dir = "./raw_data", output_file = "output.tsv",
+                 sentence_boundary_char = '।'):
+        """
+        RawData class is used for cleaning and transforming the raw data collected 
+        into a tsv file that can be fed to lasertagger.
+        input_dir : where the colleted data files are
+        output_file : where the final data should be written to
+        sentence_boundary_char : the sentence boundary character for the language 
+        """
+        self.input_dir_ = input_dir
+        self.sentence_boundary_ = sentence_boundary_char
+        self.output_file_ = output_file
+
+    def clean_initial(self, raw_line):
+        """
+        some preliminary cleaning of the data obtained by webscraping.
+        the cleaning is not exhaustive since most of the data gets discarded in the further steps.
+
+        Lines without the corresponding added and deleted lines are discarded
+        References are removed, along with stray formatting tags.
+
+        warning: the api returns data in slightly different formats for 
```

Comment:
if this is the case, can we add it in the readme file ?

Reply:
I'm sorry, what exactly should i add?

Reply:
Something like, 'different languages have different formats. so, when using the code for a new language, please modify the cleaning rules according to the patterns you observe"